csm_web/scheduler/management/commands/patchattendances.py

```diff
@@ -18,14 +18,10 @@ def handle(self, *args, **options):
             week_start = today - timedelta(days=today.weekday())
             week_end = today + timedelta(days=7)
             for profile in students:
```

Comment:
```suggestion
            for student in students:
```
Don't forget to change throughout loop body